README.md

```diff
@@ -277,6 +277,20 @@ await Promise.all([
 
 Check out the [Clustering Guide](./docs/clustering.md) when using Node Redis to connect to a Redis Cluster.
 
+### Events
+
+Node Redis client class extends Node.js's `EventEmitter` class, and the client emits an event each time the network status changes:
```

Comment:
The Node Redis client class is an `EventEmitter` and it emits an event each time the network status changes: